### ISSUE TITLE
Use numbers to represent number of changes

### ISSUE THREAD

**Issue by iilyak** [reujab/silver#4]
This is mostly a feature request not really a bug. IMO it would be better to use integers to represent number of changes from upstream. I work with codebases which can diverge significantly from upstream. Currently number of stashed changes is represented by `n` stash icons. Where `n` is number of changes ([` stashes += &icons::get("stash");`](https://github.com/reujab/silver/blob/master/src/modules/git.rs#L22)). Same goes for [`ahead` and `behind`](https://github.com/reujab/silver/blob/master/src/modules/git.rs#L36:L37)
```
graph = icons::get("ahead").repeat(ahead)
+ &icons::get("behind").repeat(behind);
```   

The problem with this approach is that it takes the screen space. It makes prompt taking few lines for some edge cases. It could be changed to show this info in colored sections like follows:
```
------\-----------------\----------------\------------------\
       > :stash_icon: 5  > :ahead_icon: 8 > :behind_icon: 10 >
------/-----------------/----------------/------------------/
```

**Comment by iilyak** [comment]
The cosmetics which could be used are:
- `nf-mdi-numeric_9_plus_box_multiple_outline` if having 9+ instead of exact number is acceptable
- `:nf-pl-left_hard_divider: :stash: 5 :nf-ple-flame_thick: :ahead: 8 :nf-ple-flame_thick: :behind: 10 :nf-pl-left_hard_divider:`
- `:nf-pl-left_hard_divider: :stash: 5 :nf-pl-left_soft_divider: :ahead: 8 :nf-pl-left_soft_divider: :behind: 10 :nf-pl-left_hard_divider:`

**Comment by reujab** [comment]
I just released v1.0.4, which fixed this issue. No more than five icons should be displayed at a time.

**Comment by iilyak** [comment]
Thank you very much.
